qlib/kernel/SignalDef.rs

```diff
@@ -856,7 +893,8 @@ pub struct SignalStruct {
     trapStopPending: bool,
 }
 
-// https://elixir.bootlin.com/linux/latest/source/arch/x86/include/uapi/asm/signal.h#L132
+// Based on linux's 'struct signalstack' in signal.h
```

Comment:
in this case please update the link :D  If they are the same, keep at least one.

Reply:
Added a reference.